intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -575,14 +575,14 @@ \subsection{Getting cluster state}
                                 |   o other (R, *, ...)    : 0   |
 
 Node type:
- ppn=36, physmem=751GB, swap=0GB, vmem=751GB, local disk=0GB
+ ppn=36, mem=751GB
```

Comment:
Where the mem info coming from?
If I run it on login node, with victini
```
....
Node type:
 ppn=36, mem=92GB
[vsc43020@gligar04 ~]$ qsub -I -l nodes=1:ppn=36,mem=92Gb
salloc: error: Job submit/allocate failed: Memory required by task is not available
```

Reply:
afaik pyslurm